perfkitbenchmarker/linux_packages/cassandra.py

```diff
@@ -90,6 +94,27 @@ def AptInstall(vm):
   _Install(vm)
 
 
+def JujuInstall(vm):
+  if vm.isController is True:
+    if vm.vm_group_specs and 'workers' in vm.vm_group_specs:
+      vm.JujuDeploy('cs:trusty/cassandra',
+                    vm.vm_group_specs['workers'].vm_count,
+                    'workers')
```

Comment:
Ah - I hadn't really noticed that you are referencing both charms on a single line... yeah that was the idea but the relate makes it more difficult. What exactly is the relate doing? The benchmark already references the ip addresses of the workers when the stress tool is run - why is the additional "add-relation" step needed?